### DISCUSSION TITLE
Problem with email verification

### DISCUSSION THREAD

**Discussion by mbalint82** [RosettaCommons/rosetta#240] [ROSIE] [answered]
Hi, I get the "Your GitHub account does not have verified academic email address" error message. My email is balint.meszaros(at)stjude.org
Thank you!

**Comment by lyskov** [comment, marked as answer]
— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.

**Comment by mbalint82** [comment]
Thank you for solving this issue
